### PR TITLE
OutputMixing module implementation (and also SendToActuators interface renaming)

### PR DESCRIPTION
**Pull request by Fberbari**
2 commits in this PR. The earlier one just renames the "sendToActuators" module to "SendInstructionsToSafety". I could have have sworn I did this in a previous PR but apparently not. *Sigh*.

Second commit is the complete implementation of the OutputMixing module, along with the relevant unit tests.

### CODE REVIEW

File: Autopilot/AttitudeManager/AttitudeDatatypes.hpp [hunk 1]
```diff
@@ -11,4 +11,9 @@ typedef struct
 
 }PID_Output_t;
 
+#define L_TAIL_OUT_CHANNEL 0 // Spike has ruddervators
+#define R_TAIL_OUT_CHANNEL 1
+#define ALERON_OUT_CHANNEL 2
```
Comment (heyitslu) [inline Autopilot/AttitudeManager/AttitudeDatatypes.hpp]: Isn't it AILERON ? not actually important haha

File: Autopilot/AttitudeManager/OutputMixing.cpp [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	OutputMixing_error_t error;
+	error.errorCode = checkInputValidity(PidOutput);
+
+    PidOutput->yawPercent += PidOutput->rollPercent * ADVERSE_YAW_MIX; // mix roll rate into rudder to counter adverse yaw
+
+    channelOut[L_TAIL_OUT_CHANNEL] =  (PidOutput->yawPercent * RUDDER_PROPORTION) - (PidOutput->pitchPercent * ELEVATOR_PROPORTION); //Tail Output Left
+    channelOut[R_TAIL_OUT_CHANNEL] =  (PidOutput->yawPercent * RUDDER_PROPORTION) + (PidOutput->pitchPercent * ELEVATOR_PROPORTION); //Tail Output Right
+
+    channelOut[ALERON_OUT_CHANNEL] = PidOutput->rollPercent;
```
Comment (heyitslu) [inline Autopilot/AttitudeManager/OutputMixing.cpp]: same prob as above

**Review by heyitslu — APPROVED** [review]
Looks good - couple typos again but honestly we could keep them

**Comment by Fberbari** [comment]
Typos have been fixed, and I'm adding the spellcheck extension to my text editor lmao.

**Review by avlepage — APPROVED** [review]
lgtm!

**Pull request merged**